jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -366,9 +411,58 @@ public String getRFC6265SetCookie()
             buf.append("; Secure");
         if (_httpOnly)
             buf.append("; HttpOnly");
+        if (_sameSite != null)
+        {
+            buf.append("; SameSite=");
+            buf.append(_sameSite.toString());
```

Comment:
See comment above, use an explicit getter (or even the reference to the field) to get the attribute value.